### ISSUE TITLE
AI trade item valuation

### ISSUE THREAD

**Issue by ilteroi** [LoneGazebo/Community-Patch-DLL#946]
just saw this:

![image](https://cloud.githubusercontent.com/assets/1358966/11870949/4f4d4f52-a4cc-11e5-8356-dbf9f37cb375.png)

what's your opinion, for me the tech is worth much more than what's desired in return

**Comment by ilteroi** [comment]
also, they like to offer gpt for embassy, and the next turn embassy for gpt

**Comment by Kolaris8472** [comment]
I've been noticing some odd deals as well, but I should point out that the AI puts an extremely high value on Paper and Nay Votes on proposals they like (I've had to pay as much as 50k for one). But maybe that's the problem. 

It does feel like, compared to vanilla, AIs are making offers based more on how they value an item rather than what would be reasonable to the player. For instance I got an offer of 1 Iron for 12 Coal when I had a global monopoly on Iron, I would assume because 1 Iron was very valuable to that AI.
